ambry-store/src/main/java/com.github.ambry.store/StoreMessageReadSet.java

```diff
@@ -146,6 +148,31 @@ public void close() {
       segment.closeView();
     }
   }
+
+  /**
+   * Do data preFetch: from disk to memory buffer.
+   * @param relativeOffset the relativeOffset to start.
+   * @param size The size requested to preFetch.
+   */
+  void preFetch(long relativeOffset, long size) throws IOException, IndexOutOfBoundsException {
```

Comment:
`IndexOutOfBoundsException` is a `RuntimeException` - need not explicitly declare that it will be thrown

Reply:
fixed.